pl-PL.lproj/Menu.strings

```diff
@@ -327,7 +327,7 @@
 "Select Paragraph / Block" = "Zaznacz akapit/blok";
 
 "Delete Selected" = "Usuń zaznaczone";
-"Move Image to" = "Przenieś obraz do";
+"Rename or Move image to" = "Zmień nazwę obrazu lub przenieś go do";
```

Comment:
It is a bit too long to be displayed in context menu.

What about use translation of "Rename / Move image to"? If still too long, use "Move image to"

Reply:
Pick the one that fits:
- Przenieś obraz / zmień jego nazwę
- Przemianuj lub przenieś obraz do
- Obraz: zmień nazwę lub przenieś